backend/api/serializers/project_serializer.py

```diff
@@ -33,6 +34,24 @@ def to_representation(self, instance: Project):
         if (groups_submitted > non_empty_groups):
             non_empty_groups = groups_submitted
 
+        extra_checks_count = ExtraCheck.objects.filter(
+            project=instance
+        ).count()
```

Comment:
This can be replaced by `instance.extra_checks`.
Can potentially avoid some db calls if the result is cached.